### PR TITLE
[SNDVOL32] Fix GetDC/ReleaseDC error management

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- GetDC() is called without properly calling ReleaseDC() as required in order to releases a device context.
Similar to https://github.com/reactos/reactos/pull/2707

## Proposed changes

- Add missing ReleaseDC

### CODE REVIEW

File: base/applications/sndvol32/dialog.c [hunk 1]
```diff
@@ -408,6 +408,7 @@ LoadDialog(
             MixerWindow->baseUnit.cx = charSize.cx;
             MixerWindow->baseUnit.cy = charSize.cy;
         }
+        ReleaseDC(NULL, hDC);
```
Comment (katahiromz) [inline base/applications/sndvol32/dialog.c:411]: insert  an empty line.

Comment (KyleKatarn31) [reply base/applications/sndvol32/dialog.c:412]: ok

Comment (KyleKatarn31) [reply base/applications/sndvol32/dialog.c:412]: ok

**Comment by KyleKatarn31** [comment]
Thanks !

**Pull request merged**